apps/list.c

```diff
@@ -244,6 +244,56 @@ static void list_kdfs(void)
     sk_EVP_KDF_pop_free(kdfs, EVP_KDF_free);
 }
 
+/*
+ * RANDs
+ */
+DEFINE_STACK_OF(EVP_RAND)
+static int rand_cmp(const EVP_RAND * const *a, const EVP_RAND * const *b)
+{
+    int ret = strcasecmp(EVP_RAND_name(*a), EVP_RAND_name(*b));
+
+    if (ret == 0)
+        ret = strcmp(OSSL_PROVIDER_name(EVP_RAND_provider(*a)),
+                     OSSL_PROVIDER_name(EVP_RAND_provider(*b)));
+
+    return ret;
+}
```

Comment:
I've some work going on in #9979 that touches on this...  I'd like to get that through first if possible

Reply:
Sure, no rush here.  There is plenty of other things to be done on the RAND front.

I could remove the `list.c` changes to avoid interfering.

Reply:
I have today's work cut out, then 😉

Reply:
I can drop this out of the PR if it helps.